jfxtras-controls/src/main/java/jfxtras/internal/scene/control/skin/CalendarTextFieldSkin.java

```diff
@@ -365,7 +389,7 @@ public String getUserAgentStylesheet() {
 		};
 		lBorderPane.getStyleClass().add(this.getClass().getSimpleName() + "_popup");
 		lBorderPane.setCenter(calendarPicker);
-		calendarPicker.showTimeProperty().set( getSkinnable().getShowTime() );
+		calendarPicker.showTimeProperty().bind(getSkinnable().showTimeProperty());
```

Comment:
I did that because I extract the popup into a global variable, but since I will instantiate it on demand (see comment above), I'll revert to the old way.